packages/web-components/src/components/va-memorable-date/va-memorable-date.scss

```diff
@@ -0,0 +1,48 @@
+@forward 'settings';
+
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-fieldset/src/styles/usa-fieldset';
+@use 'usa-legend/src/styles/usa-legend';
+@use 'usa-form-group/src/styles/usa-form-group';
+@use 'usa-hint/src/styles/usa-hint';
+@use 'usa-error-message/src/styles/usa-error-message';
+@use 'usa-memorable-date/src/styles/usa-memorable-date';
+
+@import '../va-date/va-date.css';
+@import '../../mixins/form-field-error.css';
+@import '../../global/formation_overrides';
+
```

Comment:
A bit nitpicky here but just to make it consistent with the other components that have the v3 variation, can you separate the original imports and styles along with a comment? Like we did for va-text-input:

https://github.com/department-of-veterans-affairs/component-library/blob/ceedcd6dfcdf18a84f0ec43bdc9fa05f97c410bd/packages/web-components/src/components/va-text-input/va-text-input.scss#L35-L45

Reply:
sounds good